src/dune_rules/ml_sources.ml

```diff
@@ -53,41 +54,95 @@ module Modules = struct
     }
 
   let make { libraries = libs; executables = exes; melange_emits = emits } =
-    let libraries =
+    let libraries, libs =
```

Comment:
The only reason why I have to carry over `libs` here (and `exes` and `emits` below) is because otherwise the code that checks for the "Module %S is used in several stanzas:" errors will still be triggered for cases where two libs have the same name but on of them is disabled.